packages/app/test/cypress/integration/30-search/search.spec.ts

```diff
@@ -68,3 +68,162 @@ context('Access to legacy private pages', () => {
   });
 
 });
+
+context('Search all pages', () => {
+  const ssPrefix = 'search-all-pages-';
+
+  beforeEach(() => {
+    // login
+    cy.fixture("user-admin.json").then(user => {
+      cy.login(user.username, user.password);
+    });
+    // collapse sidebar
+    cy.collapseSidebar(true);
+  });
+
+  it(`Search all pages by word is successfully loaded`, () => {
+    const searchText = 'help';
+    cy.visit('/');
+    cy.get('.rbt-input').click();
+    cy.screenshot(`${ssPrefix}search-input-focused`, { capture: 'viewport'});
+    cy.get('.rbt-input-main').type(`${searchText}`);
+    cy.screenshot(`${ssPrefix}insert-search-text`, { capture: 'viewport'});
+    cy.get('.rbt-input-main').type('{enter}');
+    cy.screenshot(`${ssPrefix}press-enter`, { capture: 'viewport'});
+
+    cy.getByTestid('search-result-base').should('be.visible');
+    cy.getByTestid('search-result-list').should('be.visible');
```

Comment:
I think adding tag(s) handling is needed before searching pages.

please check the visual difference
<img width="809" alt="Screen Shot 2022-06-10 at 16 33 25" src="https://user-images.githubusercontent.com/59536731/173014388-a29f8d10-7432-4e84-bae4-b38468cdce35.png">


<img width="781" alt="Screen Shot 2022-06-10 at 16 30 21" src="https://user-images.githubusercontent.com/59536731/173014189-6b70c1b1-9375-4879-8e23-472cc15b262a.png">

Reply:
Added new tag handling before search by tags